.github/workflows/release-snapshot.yml

```diff
@@ -5,10 +5,6 @@ on:
     branches:
       - 'v3.*'
   workflow_dispatch:
```

Comment:
Isn't this an empty key now?

Reply:
[This is the expected syntax apparently](https://github.com/orgs/community/discussions/26098#discussioncomment-3250374).